test/examples/test-zeus.good.ts

```diff
@@ -46,18 +46,22 @@ let tq = query(q => [
   ]),
 ])
 
-let tqString = `query ($cid: String, $cids: [String!]!, $cid2: String, $cids2: [String!]!) {
+let tqString = `
+fragment Card_190nlxk on Card {
+  Attack
+  def: Defense
+}
+
+query ($cid: String, $cids: [String!]!, $cid2: String, $cids2: [String!]!) {
   cardById(cardId: $cid) {
-    Attack
-    def: Defense
+    ...Card_190nlxk
     attack(cardID: $cids) {
       Attack
       Defense
     }
   }
   second: cardById(cardId: $cid2) {
-    Attack
-    def: Defense
+    ...Card_190nlxk
     attack(cardID: $cids2) {
```

Comment:
This describes what the change effectively does i.e. instead of inlining "fragments" it generates real fragments to send alongside the query.